src/main/kotlin/com/github/mitchellolsthoorn/testgenie/settings/TestGenieSettingsState.kt

```diff
@@ -0,0 +1,7 @@
+package com.github.mitchellolsthoorn.testgenie.settings
+
+/**
+ * This class is the actual data class that stores the values of the Settings entries.
+ */
+data class TestGenieSettingsState
+    constructor(var globalTimeout: String = "60", var showCoverage: Boolean = false)
```

Comment:
Okay, I am not exactly sure what this GlobalTimeout is or what it's needed for. Could you please clarify, thank you.

Reply:
I don't really know myself yet. We will look into it when we start dealing with parameters. For now, i just needed a placeholder for the template, so i just used a random EvoSuite parameter